pkgs/development/libraries/godot_headers/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub }:
+stdenv.mkDerivation rec {
+  name = "godot_headers";
+  version = "51bca3bf5d917341f3e15076c5a9191f8a5118ae";
```

Comment:
Please provide a valid version string; see e.g. `parseDrvName`. It's common to use the rev date for unreleased versions.

Reply:
Done!